### ISSUE TITLE
InlineIf issue when affecting variable ELSE is sticked to keyword

### ISSUE THREAD

**Issue by jchabrol-a3bc** [MarketSquare/robotframework-tidy#548]
With robotidy 4.4.0 option "InlineIf:line_length=140"
Before : -    ${firstname}=  IF  "${firstname}" == "${EMPTY}"  First Name  ELSE  Set Variable  ${firstname}
After: +    ${firstname}=  IF  "${firstname}" == "${EMPTY}"  First NameELSE  Set Variable  ${firstname}

**Comment by bhirsz** [comment]
Thanks for the report. It sounds like the issue we had some time ago - I will check why there is regression.

**Comment by bhirsz** [comment]
In my tests it works with 4.4.0:

```
-     ${firstname}=  IF "${firstname}" == "${EMPTY}"  First Name  ELSE  Set Variable  ${firstname}
+    ${firstname}=    IF "${firstname}" == "${EMPTY}"    First Name    ELSE    Set Variable    ${firstname}
```

Hovewer I put extra spaces to your example (ie ``IF "${firstname}" == "${EMPTY}" First Name`` to ``IF "${firstname}" == "${EMPTY}"  First Name`` so First Name is considered as keyword and not part of the condition).

Do you have any extra examples to help me reproduce it? Maybe the code is inside some structure (another IF, FOR etc)?

**Comment by jchabrol-a3bc** [comment]
Hi sory for the delay.
So first here is the whole configuration I use:

```
 [tool.robotidy]
overwrite = true
diff = true
spacecount = 2
indent = 4
continuation-indent = 4
configure = [
    "AlignSettingsSection:enabled=True",
    "AlignKeywordsSection:enabled=True:handle_too_long=compact_overflow:widths=0",
    "AlignTestCasesSection:enabled=True:handle_too_long=compact_overflow",
    "AlignTemplatedTestCases:enabled=True:min_width=5",
    "NormalizeNewLines:consecutive_lines=1:separate_templated_tests=True",
    "NormalizeSeparators:flatten_lines=True:align_new_line=True",
    "SplitTooLongLine:line_length=140",
    "RenameVariables:enabled=True:unknown_variables_case=upper:convert_camel_case=False"
] 
```

Code Before robotidy launch:
```robotframework
Create A User Payload Ready To Signup
    [Documentation]  Create User attributes with random or set with arguments parameter
    ...    Returns a dictionnary suitable to be set to /signup.
    ...    Return User parameter dictionnary
    [Arguments]
    ...    ${firstname}=${EMPTY}
    ...    ${lastname}=${EMPTY}
    ...    ${email}=${EMPTY}
    ...    ${phone_number}=${EMPTY}
    ...    ${external_id}=${EMPTY}
    ...    ${birthplace}=${EMPTY}
    ...    ${birthdate}=${EMPTY}
    ...    ${gender}=${EMPTY}
    ...    ${language_preferences}=${EMPTY}
    ...    ${profile}=${EMPTY}
    ...    ${status}=${EMPTY}
    # robocop: enable


    IF  ${{'$firstname == ""'}}
        ${firstname}=   First Name                                                                                                                      
    ELSE
        ${firstname}=   Set Variable    ${firstname}
    END
``` 
Step to reproduce:

Launch robotidy 
```robotframework
    ${firstname}=  IF  ${{'$firstname == ""'}}  First Name  ELSE  Set Variable  ${firstname}
``` 
After the 1rst launch the robotframework code is well done.

Launch robotidy again
```robotframework
    ${firstname}=  IF  ${{'$firstname == ""'}}  First NameELSE  Set Variable  ${firstname}

``` 

After the second launch the ELSE is stick to the keyword First Name (faker keyword).

So the issue occur after 2 robotidy launch.

**Comment by bhirsz** [comment]
Thanks for the detailed information, I can reproduce it - I will fix it.

**Comment by bhirsz** [comment]
It's ``NormalizeSeparators`` fault. It looks like it's removing trailing whitespace from the keyword call
